frontend/src/containers/Home.tsx

```diff
@@ -67,8 +54,8 @@ function Home() {
             onSubmit={onSearch}
             size="big"
             onClear={onClear}
-            query={filter}
```

Comment:
looks like filter is not used anymore? If this is the case, you may want to clean up the state declaration.